### PR TITLE
Add options to configure entity-cards

### LINKED ISSUE #10: Setting to hide devices and/or entities
**Is your feature request related to a problem? Please describe.**
I wanted to "hide" certain devices and/or entities from showing up on the generated dashboard.

**Describe the solution you'd like**
Add a setting with a list of entity/device ids that get excluded from showing up on just the mushroom strategy dashboard.

**Describe alternatives you've considered**
There's the setting to "hide" entities available through HASS, but this hides it globally.

**Additional context**
Discussed here: https://github.com/AalianKhan/mushroom-strategy/issues/8#issuecomment-1502174783

### PR DESCRIPTION
**Pull request by DigiLive**
Closes #10.
- Option to set the type of entity card.
- Option to hide an entity card from the dashboard.

### CODE REVIEW

File: src/mushroom-strategy.js [hunk 1]
```diff
@@ -110,7 +106,7 @@ class MushroomStrategy {
             // Create a Title card for the current domain.
```
Comment (AalianKhan) [inline src/mushroom-strategy.js:106]: If entities length is 1 but that entity is hidden via card options, the length is still 1 thus still creating a title card but with no entity cards.

**Review by AalianKhan — CHANGES_REQUESTED** [review]
Looks good 👍
I see that when I define a card in card options, it merges the default card config and my custom card config. Would it be better if it discards the default config and only uses the custom config?

**Comment by DigiLive** [comment]
> Looks good 👍 I see that when I define a card in card options, it merges the default card config and my custom card config. Would it be better if it discards the default config and only uses the custom config?

That was how it was before, but this means users will run in trouble when they forget mandatory properties (for example).
IMHO this way is more fail-save.

File: src/mushroom-strategy.js (hunk 1 above)
Comment (DigiLive) [reply src/mushroom-strategy.js:106]: Yes, I'm aware of it.
The purpose of placing the PR was making you aware I'm working on something en to review the changes in the configuration.

**Comment by ajagnanan** [comment]
Any eta for this one guys?

**Comment by DigiLive** [comment]
> Any eta for this one guys?

I can't finish up the PR without approval of @AalianKhan since reviews are required.

@AalianKhan Could you please review or change the requirement so I can merge the PR and create an alpha.2 pre-release?

**Review by AalianKhan — APPROVED** [review]

**Comment by AalianKhan** [comment]
Sorry about that, was busy with life for a few days.

**Pull request merged**